lib/hyrax/errors.rb

```diff
@@ -8,4 +8,6 @@ class MissingWorkflowError < HyraxError; end
   class WorkflowAuthorizationException < HyraxError; end
 
   class SingleUseError < HyraxError; end
+
+  class ObjectNotFoundError < HyraxError; end
```

Comment:
I think this needs to subclass `ActiveFedora::ObjectNotFoundError` so existing rescues continue to work?

How do we envision handling `Valkyrie::ObjectNotFoundError`?

Reply:
@no-reply Per your review comment above, I've added the changes to subclass ActiveFedora::ObjectNotFoundError and envision handling Valkyrie::ObjectNotFoundError. Could you review it? Thanks.